modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/configuration/admin/service/OAuth2InAssertionManagedServiceFactory.java

```diff
@@ -41,16 +42,14 @@
 import org.osgi.service.cm.ManagedServiceFactory;
 import org.osgi.service.component.annotations.Activate;
 import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
 
 /**
  * @author Arthur Chan
  */
 @Component(
 	property = Constants.SERVICE_PID + "=com.liferay.oauth2.provider.rest.internal.configuration.OAuth2InAssertionConfiguration",
-	service = {
-		ManagedServiceFactory.class,
-		OAuth2InAssertionManagedServiceFactory.class
-	}
```

Comment:
@RenanUmV @MrJohn1911 inline this class into the `LiferayJWTBearerGrantHandler`. Thx